leavestracker/apps/leaves/constants.py

```diff
@@ -0,0 +1,8 @@
+DEFAULT_ERROR="Something went wrong!"
+LEAVE_SUCCESS_MSG="your leave has been added successfully"
+DUPLICATE_LEAVE_RAISE_ERROR="Leave Already Exist"
+DUPLICATE_LEAVE_ERROR="{'non_field_errors': [ErrorDetail(string='Leave Already Exist', code='invalid')]}"
+DUPLICATE_LEAVE_MSG="some days of your leaves are already added by you"

```

Comment:
This message creates confusion If I have added one days leave. can we reframe it to - 'Leave for the date already exists'

Reply:
Can we resolve the above comments ?